package.json

```diff
@@ -14,6 +14,7 @@
     "node": ">=6"
   },
   "scripts": {
+    "lint:fix": "eslint --ignore-path .gitignore --fix .",
```

Comment:
Have you considered re-using the cli args from `lint` script via `npm run lint --fix`?